tests/api/utils/request.js

```diff
@@ -99,12 +100,38 @@ const bearerTokenReq = (token, method, endpoint, body) => {
   })
 }
 
+const signedReq = async ({ method, endpoint, url, keyUrl, privateKey, body }) => {
```

Comment:
As suggested above in the review of the `actor` tests, most consumers of `signedReq` just want to make a request from some valid fediverse user, so having default values for `keyUrl` and `privateKey` would be convenient
```suggestion
let someFediverseUser
const signedReq = async ({ method, endpoint, url, keyUrl, privateKey, body }) => {
  if (!keyUrl) {
    someFediverseUser = someFediverseUser || await createUserOnFediverse()
    keyUrl = someFediverseUser.keyUrl
    privateKey = someFediverseUser.privateKey
  }
```